apps/golden-sample-app/src/app/user-accounts/user-accounts-view/user-accounts-view.component.html

```diff
@@ -0,0 +1,9 @@
+<div *ngIf="arrangements$ | async as arrangments">
+	<h1>There are {{ arrangments.length }} accounts</h1>
```

Comment:
in our best practices we include i18n to all the static text

Reply:
Fixed, thanks